docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| keybinding         | The key or key combination that causes some functions to occur.                                                                                                                                                                                                                                                                             |
+| labels             | The labels layer is an image where pixel values are integers. Each unique integer represents a unique object, feature or classification.                                                                                                                                                                                                    |
+| layer              | Layers are the viewable objects that can be added to napari viewer. Currently napari supports the following layers: image, labels, points, shapes, surface, tracks, vectors.                                                                                                                                                                |
+| mousebinding       | The specific mouse click that causes some functions to occur.                                                                                                                                                                                                                                                                               |
+| plugin             | A napari plugin is a specialized Python package which extends napari's functionality through additional file i/o support, image analysis, or other utilities. Plugin developers specify this functionality through a combination of Python entry points and a specialized manifest file. Plugins for napari can be found at napari-hub.org. |
+| shapes             | The shapes layer allows you to display a list of NxD arrays, where each array corresponds to one shape, specified by N points in D coordinates. Currently supported shapes: rectangle, ellipses, polygons, paths, and lines. Can be used as a visualization tool to highlight ROI, or be used for downstream operations.                    |
+| surface            | A surface comprises a set of triangles (typically in three dimensions) that are connected by their common edges or corners to form the mesh.                                                                                                                                                                                                |
+| widget             | A small, composable graphical user interface . A widget could be a simple one parameter/control (e.g. a slider to adjust contrast limits, a drop-down box to select a mode) or a collection of multiple controls with data output.                                                                                                          |
```

Comment:
Definition looks good to me

Reply:
>A small, composable graphical user interface component. A widget could be a simple one parameter/control (e.g. a slider to adjust contrast limits, a drop-down box to select a mode) or a collection of multiple controls with data output.

The word **component** might be missing?